modules/mux/targets/host/test/UnitCL/host_clGetDeviceInfo.cpp

```diff
@@ -55,6 +55,12 @@ TEST_F(host_clGetDeviceInfoTest, Name) {
   const char *arch = "x86";
 #elif defined(__x86_64__) || defined(_M_X64)
   const char *arch = "x86_64";
+#elif defined(__riscv) || defined(__riscv__)
+#if __riscv_xlen == 64
+  const char *arch = "riscv64";
+#elif __riscv_xlen == 32
+  const char *arch = "riscv32";
```

Comment:
Could probably do with an `#else #error` in case someone fudged a 128-bit riscv or something